### PR TITLE
Add reconnect shortcut

### LINKED ISSUE #97: Reconnect (to last server) shortcut

### PR DESCRIPTION
**Pull request by tobibodamer**
Closes #97

### CODE REVIEW

File: H2MLauncher.UI/GlobalResources.xaml [hunk 1]
```diff
@@ -137,6 +137,54 @@
...
+
+                        <Trigger Property="IsEnabled" Value="False">
+                            <Setter Property="Opacity" Value="0.7" />
+                            <Setter Property="Foreground" Value="#CCC" />
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+        <!--<Style.Triggers>
```
Comment (metalglove) [inline H2MLauncher.UI/GlobalResources.xaml:181]: Comment?

File: H2MLauncher.UI/View/ShortcutTextBox.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,109 @@
+﻿using System.Windows.Controls;
+using System.Windows.Input;
+
+using H2MLauncher.UI.ViewModels;
+
+namespace H2MLauncher.UI.View.Controls
+{
+    /// <summary>
+    /// Interaktionslogik für ShortcutTextBox.xaml
```
Comment (metalglove) [inline H2MLauncher.UI/View/ShortcutTextBox.xaml.cs:9]: hmmm

File: H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs [hunk 3]
```diff
@@ -63,6 +65,10 @@ public partial class ServerBrowserViewModel : ObservableObject
     [NotifyPropertyChangedFor(nameof(IsRecentsSelected))]
     private IServerTabViewModel _selectedTab;
 
+    [ObservableProperty]
+    private ServerViewModel? _lastServer = null;
+    private SecureString? _lastServerPassword = null;
```
Comment (metalglove) [inline H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs:70]: The thing about making the password thing secure, is that the moment it is entered in the textbox and send as command to the application it is visible in plain text in the console.

**Review by metalglove — APPROVED** [review]
Looks good, check comments

File: H2MLauncher.UI/View/ShortcutTextBox.xaml.cs (hunk 2 above)
Comment (tobibodamer) [reply H2MLauncher.UI/View/ShortcutTextBox.xaml.cs:9]: Damn i thought i catch it this time

File: H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs (hunk 3 above)
Comment (tobibodamer) [reply H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs:70]: Yeah true, doesn't help much. If any, makes it a little harder to get it, but then, it is not really super secret. Do you want me to remove it or is it fine to leave anyways?

**Review by metalglove — APPROVED** [review]

**Pull request merged**